phc/base_client.py

```diff
@@ -243,7 +243,7 @@ def _get_user_agent():
             e.g. 'Python/3.6.7 phc-sdk-py/2.0.0 Darwin/17.7.0'
         """
         # __name__ returns all classes, we only want the client
-        client = f"phc-sdk-py/{ver.__version__}"
+        client = f"phc-sdk-py/{metadata.version(__package__)}"
```

Comment:
This solution comes from https://github.com/python-poetry/poetry/issues/273#issuecomment-1103812336

and since python 3.7 is dropped out of support list, it doesn't need to use `importlib_metadata`

Reply:
see https://github.com/lifeomic/phc-sdk-py/pull/191 to drop python 3.7 support

Reply:
Very cool I didn't know about `importlib.metadata`.